internal/builder/embedded/util.go

```diff
@@ -29,6 +31,46 @@ func solveRequestWithContext(sessionID string, image string, opts *config.BuildO
 		},
 	}
 
+	if cacheImageLayers {
+		imageName, err := reference.ParseNormalizedNamed(image)
+		if err != nil {
+			return nil, err
+		}
+		cacheTaggedName, err := reference.WithTag(imageName, "buildcache")
```

Comment:
constant for the `buildcache` literal? I would almost ask if it should be configurable, but I feel like that's premature.

Reply:
A const makes sense to me. Since this is sort of an "internal config" (i.e. details are not surfaced other than enabling layer caching) I think setting a standard name makes it easy for us to GC these cache tags if we need to in the future.

I'm trying to think of a use-case where and reason why a user would want to set a name (other than they use "buildcache"?) and nothing comes to mind.